src/express-server/config/strategies.ts

```diff
@@ -0,0 +1,44 @@
+import passport from "passport";
+import { Strategy as FacebookStrategy } from "passport-facebook";
+import { Strategy as GoogleStrategy } from "passport-google-oauth2";
+import { Strategy as LocalStrategy } from "passport-local";
+import { Strategy as TwitterStrategy } from "passport-twitter";
+import config from "../config/config.json";
+import { ErrorCodes, ErrorMessages } from "../constants/index.js";
+import { errorResponse } from "../helpers/responses.js";
+
+export const useStrategies = () => {
+  passport.serializeUser((user, done) => done(undefined, user));
+  passport.deserializeUser((user, done) => done(undefined, user));
+
+  passport.use("local", new LocalStrategy({
+    usernameField: "email",
+    passwordField: "password",
+    passReqToCallback: true,
+  }, (req, email, password, done) => {
```

Comment:
Just for future - code splitting is really cool, don't neglect with it =) (you may not refactor it here)